chapters/classes.tex

```diff
@@ -344,18 +344,14 @@ \subsection{Component Variability Prefixes discrete, parameter, constant}\label{
 \item
   A variable \lstinline!vc! declared with the \lstinline!parameter!\indexinline{parameter} prefix remains constant during transient analysis, with a value determined by the initialization problem.
 \item
-  A \emph{discrete-time} variable \lstinline!vd! has a vanishing time derivative between events.
-  Note that this is not the same as saying that \lstinline!der(vd)=0! almost everywhere,
-  as the derivative is not even defined at the events, and it is not legal
-  to apply \lstinline!der! to discrete-time variables as they are not continuous. During transient analysis the variable
-  can only change its value at event
-  instants (see \cref{events-and-synchronization}).
+  A \emph{discrete-time} variable \lstinline!vd! is a variable declared with the \lstinline!discrete!\indexinline{discrete} prefix, or a variable without any prefix that is not a continuous-time variable.
+  It has a vanishing time derivative between events.
+  Note that this is not the same as saying that \lstinline!der(vd)=0! almost everywhere, as the derivative is not even defined at the events, and it is not legal to apply \lstinline!der! to discrete-time variables as they are not continuous.
+  During transient analysis the variable can only change its value at event instants (see \cref{events-and-synchronization}).
 \item
-  A \emph{continuous-time} variable \lstinline!vn! may have a non-vanishing time
-  derivative (\lstinline!der(vn)<>0! possible) and may also
-  change its value discontinuously at any time during transient analysis
-  (see \cref{events-and-synchronization}). If there are any discontinuities the variable is
-  not differentiable.
+  A \emph{continuous-time} variable is a \lstinline!Real! variable without any prefix that is not assigned in a \lstinline!when!-clause.
```

Comment:
Alternatively, making the mutual exclusion of variable variabilities even more clear:
```suggestion
  A \emph{continuous-time} variable is a \lstinline!Real! variable without any prefix that is not a \emph{discrete-time} variable.
```

Reply:
I would prefer the current formulation, as it is more direct and the mutual exclusion seems to run the risk of circular dependency depending on how we define discrete-time variable.

Reply:
Sure, we must ensure this is aligned with the definition of _discrete-time variable_.

Reply:
Looks good now.  Resolving.